website/src/releases/2022-09/readable-page-title.mdx

```diff
@@ -0,0 +1,13 @@
+---
+date: 2022-09-29
+title: Human-readable page titles
+description: Introducing more granular and readable document's titles for all Merchant Center pages to help users identifying the context of the page.
+type: feature
+topics:
+  - Merchant Center
+  - UI Components
+---
+
+Custom Applications use a more readable document's title by default, which can be customized. You can read more about this feature in the [`Human-readable page title`](/development/human-readable-page-title) documentation.
+
+As always, if you have questions or feedback you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```

Comment:
Custom Applications now use a more readable, customizable document title by default. You can read more about this feature in the [`Human-readable page title`](/development/human-readable-page-title) documentation.

As always, if you have questions or feedback, you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).